logging-manager.js

```diff
@@ -72,9 +72,9 @@ const Logger = (module.exports = {
     return res.text()
   },
 
-  initializeErrorReporting(sentryDsn, options) {
-    const raven = require('raven')
-    this.raven = new raven.Client(sentryDsn, options)
+  initializeErrorReporting(dsn, options) {
```

Comment:
The properties of the `options` object has changed a bit between versions, but the important properties (`release`, `environment` etc) have stayed the same.